canonical-web-and-design/maas-ui
# fix(ui): fix redirect when adding a license key

## Done

- Fix redirecting to the license key list when a key is successfully added.
- Fix loading osinfo when loading the list.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a windows image (https://github.com/canonical-web-and-design/maas-ui/blob/master/HACKING.md#creating-a-fake-windows-image).
- Go to add license key form in settings and add a key.
- It should redirect you to the list of keys.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1787.
